### PR TITLE
AMP-109903 Update docs for data permission restrictions

### PR DESCRIPTION
**Pull request by cyhk**
This adds a section in _Manage your Amplitude Data settings_ about the new Permissions Tab and its functionality.

### CODE REVIEW

File: content/collections/data/en/amplitude-data-settings.md [hunk 1]
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
 
 For any unplanned events, event properties, event property types, user properties, or user property types, you can tell Amplitude Data to either mark them as **unexpected**, or to **reject** them outright. Amplitude Data ingests any events or properties marked as unexpected and send a notification to everyone subscribed to this schema. If you choose to reject unexpected data, however, Amplitude Data doesn't ingest or store the rejected data. Subscribers still receive a notification.
 
-Click *Save* to implement any changes you make to your schema settings.
\ No newline at end of file
+Click _Save_ to implement any changes you make to your schema settings.
+
+## The Permissions Tab
```
Comment (markzegarelli) [inline content/collections/data/en/amplitude-data-settings.md:98]:
```suggestion
## The Permissions tab
```

File: content/collections/data/en/amplitude-data-settings.md [hunk 2]
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
...
+
+Your company may want to limit the number of people who have access to perform Data Management functions. For example, you may want most of your users to have the Member role so they can create dashboards and charts but prefer that custom event creation and other data management features be limited to managers or administrators.
```
Comment (markzegarelli) [inline content/collections/data/en/amplitude-data-settings.md:100]:
```suggestion
You may want to limit the number of people who have access to perform Data Management functions. For example, if you want most of your users to have the Member role so they can create dashboards and charts but limit custom event creation and other data management features to managers or administrators.
```

File: content/collections/data/en/amplitude-data-settings.md [hunk 3]
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
...
+
+In the permissions tab, you can add additional restrictions for selected roles, limiting data management access while keeping other permissions for those roles.
+
+Under Restrict access in Amplitude Data, you can specify from the following options to restrict permissions to the viewer level on a per-project basis:
```
Comment (markzegarelli) [inline content/collections/data/en/amplitude-data-settings.md:104]:
```suggestion
Under Restrict access in Amplitude Data, specify from the following options to restrict permissions to the viewer level on a per-project basis:
```
@cyhk Is this in Data settings? Will it be a new tab?

File: content/collections/data/en/amplitude-data-settings.md [hunk 4]
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
...
+
+- Use Default Restrictions
+- Restrict Members
+- Restrict Members and Managers
+
+Then, click _Save_.
+
+The Permissions tab is only visible to admins and permission restrictions cannot be applied to admins themselves.
```
Comment (markzegarelli) [inline content/collections/data/en/amplitude-data-settings.md:112]:
```suggestion
The Permissions tab is only visible to users with the Administrator role, and permission restrictions don't apply to Administrators.
```

File: content/collections/data/en/amplitude-data-settings.md [hunk 5]
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
...
+
+### Copy to Other Projects
```
Comment (markzegarelli) [inline content/collections/data/en/amplitude-data-settings.md:114]:
```suggestion
### Copy to other projects
```

File: content/collections/data/en/amplitude-data-settings.md [hunk 6]
```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
...
+
+You can also copy current permission restriction settings to other projects by clicking on the _Copy to Other Projects_ button. To copy settings, select the desired projects and click _Save_.
```
Comment (markzegarelli) [inline content/collections/data/en/amplitude-data-settings.md:116]:
```suggestion
Click _Copy to Other Projects_ to apply the current permission restriction settings to another project.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Review by markzegarelli — APPROVED** [review]

File: content/collections/data/en/amplitude-data-settings.md (hunk 3 above)
Comment (cyhk) [reply content/collections/data/en/amplitude-data-settings.md:104]: Yes, this will be a new tab in Data settings.

Comment (cyhk) [reply content/collections/data/en/amplitude-data-settings.md:104]: To clarify - the Permissions tab is a new tab in Data settings, Restrict access in Amplitude Data is the title of the bottom section on the page, which looks like this:
Screenshot 2024-10-18 at 16 50 34

**Comment by cyhk** [comment]
@markzegarelli - asking for a re-review so I don't slip anything past unchecked, but I made some changes based on your suggestions:
- slightly more extensive tweaking for the first two paragraphs
- removed the part about permission restrictions not applying to admins and added a part about the feature being available for enterprise customers only for the last paragraph before the *Copy to other projects* section.

**Review by markzegarelli — APPROVED** [review]

**Comment by markzegarelli** [comment]
Thanks @cyhk! I had automerge enabled on this, so it merged after I approved and resolved conversations. Is that OK?

**Pull request merged**